src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/DisputeMapper.java

```diff
@@ -14,22 +14,24 @@
 import ca.bc.gov.open.jag.tco.oracledataapi.api.model.ViolationTicket;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeCount;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeStatus;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Plea;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.ViolationTicketCount;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.YesNo;
+import ca.bc.gov.open.jag.tco.oracledataapi.repository.impl.ords.DisputeRepositoryImpl;
 
 @Mapper
 public interface DisputeMapper {
-	
+
 	DisputeMapper INSTANCE = Mappers.getMapper(DisputeMapper.class);
-	
+
 	// Map dispute data from ORDS to Oracle Data API dispute model
-	@Mapping(source = "dispute.entDtm", target = "createdTs")
+	@Mapping(source = "dispute.entDtm", target = "createdTs", dateFormat = DisputeRepositoryImpl.DATE_TIME_FORMAT)
 	@Mapping(source = "dispute.entUserId", target = "createdBy")
-	@Mapping(source = "dispute.updDtm", target = "modifiedTs")
+	@Mapping(source = "dispute.updDtm", target = "modifiedTs", dateFormat = DisputeRepositoryImpl.DATE_TIME_FORMAT)
 	@Mapping(source = "dispute.updUserId", target = "modifiedBy")
 	@Mapping(source = "dispute.disputeId", target = "disputeId")
-	@Mapping(source = "dispute.disputeStatusTypeCd", target = "status")
+	@Mapping(source = "dispute.disputeStatusTypeCd", target = "status", qualifiedByName="mapDisputeStatus")
```

Comment:
This was needed as the DisputeStatus names in ORDS are different than H2. Here, we're using a custom mapper to match on the enum's shortCd